server/src/error.rs

```diff
@@ -1,16 +1,16 @@
+use axum::http::StatusCode;
+use axum::response::{IntoResponse, Response};
+
 use bs58::decode::Error as BS58Error;
 use diesel::result::Error as DieselError;
 use r2d2::Error as R2D2Error;
 use reqwest::Error as ReqwestError;
 use serde_json::Error as JsonError;
 use std::string::FromUtf8Error;
 
-// TODO: create error structure/messages for all modules
+// TODO: rework error handling
```

Comment:
Does this TODO need to be there? That's where issues go to die is in TODO blocks in the codebase